website/guides/08_package-creation.mdx

```diff
@@ -157,7 +157,7 @@ manifests:
 ```
 
 #### 4\. Value Definitions (Optional)
-Link to the official package config [documentation](https://github.com/glasskube/glasskube/blob/main/website/docs/04_design/package-config.md)
+Link to the official package config [documentation](https://github.com/glasskube/glasskube/blob/main/website/docs/05_design/package-config.md)
```

Comment:
Since the linked file is also part of the website, we might as well link to the website: 
```suggestion
Link to the official package config [documentation](/docs/design/package-config)
```
(same for the link on line 83 in this file)